### PR TITLE
enh(ui): Fixes to Events view Listing / Details / Graph

### PR DESCRIPTION
**Pull request by bdauria**
## Description
- Always be able to search, even if current search is equal to next search
- Reduce graph padding
- Fix graph not showing negative values
- disable graph animation
- Fix active checks marks not showing for service details
- Increase graph Y Axis margin width
- Fix command line copy 

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

### CODE REVIEW

File: www/front_src/src/Resources/Details/Body/tabs/Details/DetailsCard/cards.tsx [hunk 1]
```diff
@@ -86,7 +86,10 @@ const getDetailCardLines = (
       ...getDateTimeLines({ label, field }).getLines(),
       {
         key: `${label}_active`,
-        line: details.active_checks ? <ActiveLine /> : null,
+        line:
+          details.active_checks || details.is_active_check ? (
```
Comment (kduret) [inline www/front_src/src/Resources/Details/Body/tabs/Details/DetailsCard/cards.tsx:90]: we'll need to make API consistent...

File: www/front_src/src/Resources/Details/Body/tabs/Details/index.tsx [hunk 2]
```diff
@@ -69,19 +69,19 @@ const DetailsTab = ({ details }: Props): JSX.Element => {
...
-          severity: Severity.error,
-        });
+    try {
+      copy(details.command_line);
+
+      showMessage({
+        message: labelCommandCopied,
+        severity: Severity.success,
+      });
+    } catch (_) {
```
Comment (kduret) [inline www/front_src/src/Resources/Details/Body/tabs/Details/index.tsx:79]: is underscore needed in coding style rules ?

File: www/front_src/src/Resources/Details/Body/tabs/Graph/index.tsx [hunk 3]
```diff
@@ -30,7 +29,9 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   graphContainer: {
     display: 'grid',
-    padding: theme.spacing(3),
+    paddingTop: theme.spacing(2),
+    paddingLeft: theme.spacing(1),
+    paddingRight: theme.spacing(1),
```
Comment (kduret) [inline www/front_src/src/Resources/Details/Body/tabs/Graph/index.tsx:34]: If padding bot is 0
```suggestion
    padding: theme.spacing(2, 1, 0),
```

File: www/front_src/src/Resources/Details/Body/tabs/Details/index.tsx (hunk 2 above)
Comment (bdauria) [reply www/front_src/src/Resources/Details/Body/tabs/Details/index.tsx:79]: Yes, the empty catch block is not allowed

**Pull request merged**